packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -80,6 +80,9 @@ class BuildIOSFrameworkCommand extends BuildSubCommand {
       ..addFlag('cocoapods',
         help: 'Produce a Flutter.podspec instead of an engine Flutter.xcframework (recommended if host app uses CocoaPods).',
       )
+      ..addFlag('static',
+        help: 'Build plugins as static frameworks. Link on, but do not embed these frameworks in the existing Xcode project.',
```

Comment:
nits (personal opinion):
As the command runs on a single plugin, maybe:

```suggestion
        help: 'Build the plugin as a static framework. Link on, but do not embed the framework in the existing Xcode project.',
```

Reply:
> As the command runs on a single plugin, maybe:

The command compiles all the plugins, one framework per plugin.

Reply:
Took another look and I'm a little confused. Is this flag for building a plugin project? Or it is for building a flutter app and it builds all the dependencies as static libs?